stable/fluent-bit/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: fluent-bit
-version: 0.2.13
+version: 0.2.14
```

Comment:
You're adding a feature, please bump the minor.

Reply:
fixed